lib/ansible/module_utils/basic.py

```diff
@@ -1692,13 +1692,16 @@ def _check_required_if(self, spec, param=None):
             else:
                 term = 'all'
 
-            if key in param and param[key] == val:
+            if not isinstance(val, (list, tuple)):
```

Comment:
I also believe it better to do the required_if in two line and and avoid this change:

```python
        required_if=[
            ['jump', 'TEE', ['gateway']],
            ['jump', 'tee', ['gateway']],
        ]
```